core/src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -528,7 +540,7 @@ public void copyItemsetAnswer(QuestionDef q, TreeElement targetNode, IAnswerData
         triggerTriggerables(destRef);
 
         // initialize conditions for the node (and sub-nodes)
-        initializeTriggerables(destRef);
+        initTriggerablesRootedBy(destRef);
```

Comment:
Hey,

CopyItemsetAnswer is completely unused but was included for spec compliance. When you use it, it copies the full subtree of the itemset nodeset for the selected choices. For now this is almost certainly fine, I'd probably make a note that it should possibly exclude subtrees copied over in that way, though.